### ISSUE TITLE
Task: Change Management Plan for Alt Text

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#13106]
## Description
We need to determine the change management plan and how to socialize any CMS changes that will be happening with editors.

- Create Knowledge Base articles
- Training Video around uploading images and adding alt text: Determine if this should be priority or a fast follow

**Comment by laflannery** [comment]
I just found this - I'm closing because it's a duplicate of #15549 and #15655
